flujo/validation/fixers.py

```
@@ -0,0 +1,425 @@
+    from fnmatch import fnmatch
+
+    def _allowed(rid: str) -> bool:
+        if not rules:
+            return True
+        ridu = rid.upper()
+        for pat in rules:
+            try:
+                if fnmatch(ridu, str(pat).upper()):
+                    return True
+            except Exception:
+                continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)